lib/openregister.rb

```diff
@@ -1,5 +1,8 @@
 require 'morph'
 require 'rest-client'
+require 'json'
+require 'registers_client'
+require 'register_client'
 
 module OpenRegister
   VERSION = '0.0.1' unless defined? OpenRegister::VERSION
```

Comment:
should we bump the version number and tag a release?

Reply:
I was going to bump the version number. Not sure about tagging a release. Also, the version is defined in the `gemspec` file I believe.